tablex.typ

```diff
@@ -2821,6 +2841,7 @@
     map-vlines: none,
     map-rows: none,
     map-cols: none,
+    map-order: ("rows", "cols"),
```

Comment:
I think it should just be a string, `"rows"` or `"cols"`. This design would allow running the mappers multiple or even zero times.

Reply:
So you mean, if `map-order` is `"cols"`, then it should first run the cols mapper and then the rows mapper and vice versa? Then I would choose a more descriptive name for the options such as `"rows-first"` and `"cols-first"`.